src/Schema/Events/MyCollection.ts

```diff
@@ -17,14 +19,43 @@ import { ActionType } from "."
  *  {
  *    action: "addCollectedArtwork",
  *    context_module: "myCollectionHome",
- *    context_owner_type: "myCollection"
+ *    context_owner_type: "myCollection",
+ *    platform: "mobile"
  *  }
  * ```
  */
 export interface AddCollectedArtwork {
   action: ActionType.addCollectedArtwork
   context_module: ContextModule.myCollectionHome
   context_owner_type: OwnerType.myCollection
+  platform: Platform
+}
+
+/**
+ * Schema describing 'Save Collected Artwork' events
+ * @packageDocumentation
+ */
+
+/**
+ * A user taps "save artwork" to save an artwork to My Collection.
+ *
+ * This schema describes events sent to Segment from [[saveCollectedArtwork]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "saveCollectedArtwork",
+ *    context_module: "myCollectionHome",
+ *    context_owner_type: "myCollection",
+ *    platform: "mobile"
+ *  }
+ * ```
+ */
+export interface SaveCollectedArtwork {
+  action: ActionType.saveCollectedArtwork
+  context_module: ContextModule.myCollectionHome
```

Comment:
is it correct to have the saveCollectedArtwork event context in `myCollectionHome`?
edit and delete have `myCollectionArtwork` as context